### DISCUSSION TITLE
Missing Glyphs

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#3836] [General]
With the Very recent northrend glyph introduction I have noticed that there are certain glyphs missing or certain glyphs that should not be there. Im only gonna do this for Druid but i will post it around so other classes can pitch in for them
Druid is missing 
Starfall, Mangle, Focus, Swiftmend, Rapid Rejuv.

**Comment by neifion-00000000** [comment]
Mage is missing (at least):
Fireball, Frostbolt

**Reply by Hihilist** [reply]
And Fire Blast

**Comment by heyitsbench** [comment]
Worth pointing out that Glyph of Focus has a level requirement of 70, so I doubt that'll be added.

All of the others in the thread currently are within the 60 levels we're at now.

**Reply by Never6** [reply]
savage roar and typhoon were added and they require 70+

**Reply by heyitsbench** [reply]
> savage roar and typhoon were added and they require 70+

Can they actually be equipped?

**Reply by Never6** [reply]
nah they cant be used but they are in the shop

**Reply by heyitsbench** [reply]
I don't see why they were added then 🤔

**Reply by Annamaria-CC** [reply]
didn't look at level, just added them all (well as you can see I missed a few)

**Comment by SgtRumpsteak** [comment]
Add missing warrior glyphs here pls.
For now I only noticed Glyph of Taunt is missing, wasn't able to check for more yet.
Maybe someone adds more as comment.

**Comment by Frodhi** [comment]
Glyph of Hymn of Hope should not be available to priests(the spell is not available yet)

**Comment by BeefSteaker** [comment]
**Shaman**
~~Glyph of healing wave~~
~~Glyph of Shocking~~

Missing after restart

**Comment by crusarian** [comment]
Glyph of shocking is missing for shamans

**Comment by ohsiros** [comment]
Glyph of Immolate is missing for Warlocks.

**Reply by BeefSteaker** [reply]
This one still missing

**Comment by Neversong97** [comment]
Glyph of Holy Shock is missing for Paladins PLS ASAP this is most important glyph for all Hpalas in PVP. Idk why holy pala mains didnt report it I think they should question their spec and go watch some guides
